rir/src/compiler/pir/instruction_list.h

```diff
@@ -29,7 +29,9 @@
     V(Extract2_2D)                                                             \
```

Comment:
It's not in this pr, but why are the `Extract*_1D` under binops and not rw instructions?

Reply:
I moved both

Reply:
ehm, yeah, well I guess they are not RW, but just R, that's maybe why?
Maybe check where the lists are used and just use your best judgement...